AuRoRA/test/gcc.py

```
@@ -0,0 +1,113 @@
+                
+                # Add assistant response
+                messages.append({"role": "assistant", "content": answer})
+                
+                # Callback with result
+                request.callback(answer)
+                
+                self.request_queue.task_done()
+                
+            except queue.Empty:
+                continue
+            except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)